tests/testthat/test-particle-filter.R

```diff
@@ -1518,3 +1493,53 @@ test_that("can run replicated model on gpu", {
   ll_g <- p_g$run(pars)
   expect_identical(ll_c, ll_g)
 })
+
+
+test_that("run particle filter on continuous model", {
+  dat <- example_continuous()
+  n_particles <- 42
+  p <- particle_filter$new(dat$data, dat$model, n_particles, dat$compare,
+                           index = dat$index, seed = 1L,
+                           stochastic_schedule = dat$stochastic_schedule)
+  pars <- list(init_Ih = 0.8,
+               init_Sv = 100,
+               init_Iv = 1,
+               nrates = 15)
+  res <- p$run(pars)
+  expect_is(res, "numeric")
+
+  mod <- dat$model$new(pars, 0, 1, seed = 1L)
+  len <- mod$info()$len
+
+  state <- p$state()
+  expect_is(state, "matrix")
+  #expect_equal(dim(state), c(len, n_particles))
```

Comment:
Code factor complaining here. Can remove this line, replace with the known size of the model, or wait for [this PR](https://github.com/mrc-ide/odin.dust/pull/105) to be merged and uncomment this line.